packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -2197,6 +2197,9 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
   /// set to true
   bool get impliesAppBarDismissal => hasActiveRouteBelow || _entriesImpliesAppBarDismissal > 0;
 
+  /// {@macro flutter.widgets.RawDialogRoute.fullscreenDialog}
+  bool get fullscreenDialog => false;
```

Comment:
Can you add a TODO here?
```suggestion
  /// {@macro flutter.widgets.RawDialogRoute.fullscreenDialog}
  // TODO(dkwingsmt): Rename `ModalRoute.fullscreenDialog` something semantically suitable for a modal.
  // https://github.com/flutter/flutter/issues/168949
  bool get fullscreenDialog => false;
```